edge-functions/ab-testing-statsig/pages/[bucket].tsx

```diff
@@ -68,45 +68,9 @@ function BucketPage({ bucket }: Props) {
         </Text>
         <Text>
           Buckets are statically generated at build time in a{' '}
-          <Code>/[bucket]</Code> page so its fast to rewrite to them.
+          <Code>/[bucket]</Code> page so its fast to rewrite to them. Take a
+          look at the <Code>middleware.ts</Code> file to know more.
         </Text>
```

Comment:
There is a type error on this page, you would want to add throwOnMissing also here  `const { reload } = useRouter(true)`

Reply:
Changed the PR to WIP because I'm consistently making changes until all examples are working with Next.js canary.